test/DeltaTriplesTest.cpp

```diff
@@ -0,0 +1,269 @@
+// Copyright 2023 - 2024, University of Freiburg
+//  Chair of Algorithms and Data Structures.
+//  Authors:
+//    2023 Hannah Bast <bast@cs.uni-freiburg.de>
+//    2024 Julian Mundhahs <mundhahj@tf.uni-freiburg.de>
+
+#include <gtest/gtest.h>
+
+#include "./util/GTestHelpers.h"
+#include "./util/IndexTestHelpers.h"
+#include "absl/strings/str_split.h"
+#include "engine/ExportQueryExecutionTrees.h"
+#include "index/DeltaTriples.h"
+#include "index/IndexImpl.h"
+#include "index/Permutation.h"
+#include "parser/RdfParser.h"
+
+namespace Matchers {
+inline auto InAllPermutations = [](auto InnerMatcher) {
```

Comment:
`inline` in a `.cpp` file is somewhat pointless and misleading., If you want to you can make the lambdas `constexpr` or just plain `auto`.